RepositoryCompiler/CodeModel/CaDETModel/Metrics/CaDETClassMetrics.cs

```diff
@@ -8,6 +8,10 @@ public class CaDETClassMetrics
         public int NAD { get; set; }
         public int WMC { get; set; }
         /// <summary>
+        /// ATFD: Access To Foreign Data
+        /// DOI: 10.1109/ESEM.2009.5314231
+        /// </summary>
+        public int ATFD { get; set; }
```

Comment:
I think I mislead you with the DOI, and placing it in this class.

We would want the DOI to be in the summary comment of the actual function doing the calculation, as the algorithm is what is closely tied to the DOI, not necessarily the original definition of the metric. You should still leave the full name in the summary (and maybe even add alternative names from that overview paper that we keep referencing).

You can take a look at Vladislav's MemberMetrics and functions in the Calculator to see how it should look.